frontend/public/components/utils/resource-log.jsx

```diff
@@ -114,6 +144,28 @@ export class ResourceLog extends SafetyFirst {
     saveAs(blob, `${filename}.log`);
   }
 
+  // Toggle currently displayed log content to/from fullscreen
+  _toggleFullscreen () {
+    const logConsole = document.querySelector(".log-console");
```

Comment:
Is using `document.querySelector(".log-console")` best practice?  I would think there is a better method for this.   Perhaps a [Ref](https://reactjs.org/docs/refs-and-the-dom.html).

Reply:
Much less code using that util, still curious about this one.

Reply:
hi, yes, looking into using Refs, thanks

Reply:
Hi, fixed to use Refs